src/AllArticles.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+const style = {
+  padding: '20px 35px',
+  width: '78%',
+  margin: '10px auto',
+  borderRadius: '3px',
+  fontSize: '1.1em'
+}
+
+const padding = {
+  paddingBottom: '140px'
+}
+
+export default class AllArticles extends Component {
+  render () {
+    if (this.props.markdownList === undefined) {
+      return null
+    }
+    const articles = this.props.markdownList.map((item, id) => {
+      return (
+        <div
+          style={style}
+          className='articleWrapper'
+          key={id}
+          >
+          <article
+            dangerouslySetInnerHTML={{__html: item.content}}
```

Comment:
`dangerouselySetInnerHTML` is something you should almost never do. In this application, it seems to be justified, but I hope you don't take as a lesson from this class that this is a good idea!